spotbugs-tests/src/test/java/edu/umd/cs/findbugs/classfile/MethodsCallOrderTest.java

```diff
@@ -0,0 +1,74 @@
+package edu.umd.cs.findbugs.classfile;
+
+import edu.umd.cs.findbugs.*;
+import edu.umd.cs.findbugs.ba.AnalysisContext;
+import edu.umd.cs.findbugs.ba.ClassContext;
+import edu.umd.cs.findbugs.classfile.impl.ClassFactory;
+import org.apache.bcel.classfile.Method;
+import org.junit.jupiter.api.Test;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.fail;
+
+/**
+ * @author Tomas Polesovsky
+ */
+public class MethodsCallOrderTest {
```

Comment:
In Junit Jupiter the test classes have package private visibility. Can you please remove the `public` to not overwrite this?